specification/securityinsights/resource-manager/Microsoft.SecurityInsights/stable/2020-01-01/SecurityInsights.json

```diff
@@ -1030,378 +2045,529 @@
         "value"
       ]
     },
-    "AlertSeverity": {
-      "description": "The severity of the alert",
-      "enum": [
-        "High",
-        "Medium",
-        "Low",
-        "Informational"
+    "DataConnectorTenantId": {
+      "description": "Properties data connector on tenant level.",
+      "properties": {
+        "tenantId": {
+          "description": "The tenant id to connect to, and get the data from.",
+          "type": "string"
+        }
+      },
+      "type": "object"
+    },
+    "DataConnectorWithAlertsProperties": {
+      "description": "Data connector properties.",
+      "properties": {
+        "dataTypes": {
+          "$ref": "#/definitions/AlertsDataTypeOfDataConnector",
+          "description": "The available data types for the connector."
+        }
+      },
+      "type": "object"
+    },
+    "FusionAlertRule": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/AlertRule"
+        }
       ],
-      "type": "string",
-      "x-ms-enum": {
-        "modelAsString": true,
-        "name": "AlertSeverity",
-        "values": [
-          {
-            "description": "High severity",
-            "value": "High"
-          },
-          {
-            "description": "Medium severity",
-            "value": "Medium"
-          },
-          {
-            "description": "Low severity",
-            "value": "Low"
+      "description": "Represents Fusion alert rule.",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/FusionAlertRuleProperties",
+          "description": "Fusion alert rule properties",
+          "x-ms-client-flatten": true
+        }
+      },
+      "type": "object",
+      "x-ms-discriminator-value": "Fusion"
```

Comment:
This property must be required

Reply:
the property that contaisn the discriminator must be required.  That is, if I have a property called 'type'  whose value is the discriminator value, the 'type' property must be required.

Reply:
not sure I get it - "Fusion" is not a property
If I am not mistaken the x-ms-discriminator-value defines an alias of the type
am I missing something?

Reply:
In this case, it means the discriminator from the parent type, which is, in fact required.  So, you can disregard

Reply:
Yes, fudion is the value, 'kind' is the property, and it looks like it is requred in the referenced type